table/tables/partition.go

```diff
@@ -1687,6 +1687,15 @@ func partitionedTableUpdateRecord(gctx context.Context, ctx sessionctx.Context,
 	return nil
 }
 
+func hasGlobalIndex(tblInfo *model.TableInfo) bool {
```

Comment:
Does this function be called?

Reply:
resolved. this function is no longer used, and I removed it.